app/views/course/forum/posts/_post.html.slim

```diff
@@ -21,10 +21,9 @@
       - if defined?(show_buttons)
         div.pull-right
           .btn-group
-            - if can? :reply_topic, @topic
-              = link_to reply_course_forum_topic_post_path(current_course, @forum, @topic, post),
-                        title: t('.reply'), class: ['btn', 'btn-primary'] do
-                = fa_icon 'mail-reply'.freeze
+            = link_to reply_course_forum_topic_post_path(current_course, @forum, @topic, post),
```

Comment:
We have a reply box and the bottom, this one is for replying individual posts, think they should be same state.

But both of them should be hide and new abilities need to be defined when we implement the ability of locked topic.